ocs/defaults.py

```diff
@@ -83,3 +83,50 @@
         TEMPLATE_DIR, "PersistentVolumeClaim_new.yaml"
     )
 )
+
+CSI_TEMPLATE_DIR = os.path.join(TOP_DIR, "templates/CSI/")
+
+CEPHFS_SC = load_yaml_to_dict(
```

Comment:
Is it really necessary for us to have an entry in defaults for every yaml file? This seems like overkill to me.

Reply:
There are only couple of standard yaml's , we could do in some standard for loop with file syntax name but for now it looks fine to me to do in one place.